src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/attendance/AbsentReasonsDto.kt

```diff
@@ -4,12 +4,12 @@ import io.swagger.annotations.ApiModelProperty
 import uk.gov.justice.digital.hmpps.whereabouts.model.AbsentReason
 
 data class AbsentReasonsDto(
-  @ApiModelProperty(value = "List of paid absent reasons", example = "[AcceptableAbsence]", position = 1)
+  @ApiModelProperty(value = "List of paid absent reasons", example = "[\"ApprovedCourse\"]", position = 1)
   val paidReasons: Set<AbsentReason>? = null,
 
-  @ApiModelProperty(value = "List of unpaid absent reasons", example = "[UnacceptableAbsence]", position = 2)
+  @ApiModelProperty(value = "List of unpaid absent reasons", example = "[\"RestInCellOrSick\"]", position = 2)
```

Comment:
This broke v2 swagger by producing invalid json. OpenAPI is a bit more resilient and coerces the example to a string (which is also wrong but at least valid json)